integration_tests/src/main/python/conftest.py

```diff
@@ -77,6 +77,15 @@ def is_emr_runtime():
 def is_dataproc_runtime():
     return runtime_env() == "dataproc"
 
+def get_test_tz():
+    return os.environ["PYSP_TEST_spark_sql_session_timeZone"]
+
+def is_utc():
```

Comment:
I didn't realize `is_tz_utc` existed. I think we should probably only keep one way of checking if the timezone is UTC or not. `is_tz_utc` has the problem that we need a spark session to make it work. That is fine, but it also makes it difficult to use it to skip a test unless it happens after the test starts to run. This patch removes all uses of `is_tz_utc` so perhaps we should also delete the implementation too.

Reply:
Done in https://github.com/NVIDIA/spark-rapids/pull/9482
- default value with `PYSP_TEST_spark_sql_session_timeZone`
- Remove useless `is_tz_utc`